### PR TITLE
Feature/supporting information

### PR DESCRIPTION
**Pull request by lukaszdrynski**

### CODE REVIEW

File: app/routes.js [hunk 1]
```diff
@@ -374,14 +374,53 @@ router.post('/v1/arrangements/your-ex', function(req, res) {
...
+  var whenLeaveDate = {
+    day: req.body.whenLeaveDay || 05,
+    month: req.body.whenLeaveMonth || 04,
+    year: req.body.whenLeaveYear || 2010
+  };
+
+  var separation = moment.duration(moment().diff(moment(whenLeaveDate))).years();
+
+  var separator = '/';
+  req.session.supportingInformation.dateLeave = whenLeaveDate.day + separator + whenLeaveDate.month + separator + whenLeaveDate.year;
```
Comment (sophiedrake) [inline app/routes.js]: can you do `whenLeaveDate.join('/')`?

File: app/routes.js [hunk 2]
```diff
@@ -374,14 +374,53 @@ router.post('/v1/arrangements/your-ex', function(req, res) {
...
+
+  if (separation >= 2) {
+    res.redirect('/v1/supporting-information/agree');
+  } else {
+    res.redirect('/v1/supporting-information/exit-date-fail');
```
Comment (sophiedrake) [inline app/routes.js]: maybe 'exit-date-error' might be 'nicer' than fail?

File: app/views/v1/supporting-information/agree.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{% extends "question.html" %}
+
+{% block question %}
+Does your husband/ wife agree to the ending of the marriage?
+{% endblock %}
+
+{% block form %}
+
+<fieldset class="form-group">
+    <label class="form-label">
```
Comment (sophiedrake) [inline app/views/v1/supporting-information/agree.html]: Does this need a `text` class or is it okay?

File: app/views/v1/supporting-information/when.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{% extends "question.html" %}
+
+{% block question %}
+When did you seperate ?
```
Comment (sophiedrake) [inline app/views/v1/supporting-information/when.html]: spelling: separate, don't need the space before the '?'

File: app/routes.js (hunk 1 above)
Comment (lukaszdrynski) [inline app/routes.js]: whenLeaveDate is na object, not an array

**Comment by sophiedrake** [comment]
+1

**Pull request merged**